pages/Documents/MessagingChannels/PushNotificationService/tls-authentication.md

```diff
@@ -12,7 +12,8 @@ permalink: push-notification-service-tls-authentication.html
 indicator: messaging
 ---
 
-Customers supply a valid client key and certificate for a TLS connection (one-way SSL). The key and certificate is used in every request to the relevant proxy call. The key is securely stored in LivePerson’s server.
+Customers supply a valid client key and certificate for an mutual TLS connection (two-way SSL).  
+The key and certificate is used in every request to the relevant proxy call. The key is securely stored in LivePerson’s server.
```

Comment:
Please either add or remove a line break between lines 15 and 16, depending on whether these should constitute 1 or 2 paragraphs.

Reply:
Hey @j9t in markdown a line break is a double space before the end of the line. (which I added).
I've noticed a lot of trending towards using HTML as a supplementary syntax (eg the image links). What is the standard here?

Reply:
That’s not visible here, and not even in the “raw” file.

Could you please take either approach, have a blank line between these if they are to mean 2 paragraphs, or remove the line break entirely, if this to be 1 paragraph? Thank you!